INSTALL.asciidoc

```diff
@@ -45,15 +60,27 @@ cd /tmp/os-autoinst-run
 --------------------------
 7.  create a minimal "vars.json" config file
 +
+.vars.json
+[source,json]
 ---------------------------------------------------------
-cat <<EOF >vars.json
```

Comment:
why delete this? If you keep it as an executable command it's easier to use for shell users